sim/dex.ts

```diff
@@ -138,6 +139,60 @@ const Natures: {[k: string]: Nature} = {
 };
 export const toID = Data.toID;
 
+// interface for the builder.
+interface FormatSection {
+	section: string;
+	column?: number;
+	formats: FormatData[];
+}
```

Comment:
Considering this is only used by the `merge` function, maybe it should be moved inside the function for clarity?